bundles/core/src/test/java/com/adobe/cq/wcm/core/components/internal/models/v2/PageImplTest.java

```diff
@@ -161,4 +161,18 @@ void testHtmlPageItemsConfig() {
         assertNotNull(page.getHtmlPageItems());
         assertEquals(3, page.getHtmlPageItems().size(), "Unexpected number of HTML page items");
     }
+    
+    @Test
+    void testgetCanonicalURL_External() {
```

Comment:
The not null assertion isn't enough to validate that it is functioning properly.
Need to ensure that the value returned is the value expected.

Reply:
Required changes have been taken care.

Reply:
@vladbailescu Did you get a chance to review the changes? These changes taken care upon your review comments?